lib/fog/openstack/auth/token/v3.rb

```diff
@@ -50,6 +55,7 @@ def path
           end
 
           def scope
+            return false if @application_credential
```

Comment:
It feels to me like `nil` might be a bit more appropriate than false here. It should still return falsey in the conditional, but just conceptually a nil scope makes more sense to me than a false scope. Does that make sense?

Reply:
Looks fine to me to change it to `nil`. I will test if it then still works.